### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by noobythedumbass** [Rectify11/Installer#673]
### Description

When I uninstalled Rectify11 using the installer and restarted my PC, the screen went black. I can still open apps using keyboard shortcuts, but running explorer.exe doesn't work. Can anyone help me?

### Steps To Reproduce

1. Install Rectify11 on your PC.
2. Use the installer to uninstall Rectify11.
3. Restart your PC after the uninstallation.
4. Notice that the screen is black after restarting.
5. Try opening apps using keyboard shortcuts (observe that this sometimes works).
6. Attempt to run explorer.exe (observe that it doesn't work).

### Rectify11 Version

Version: 3.2

### Windows Version

Windows 11 24H2 (I forgot the build version since I don't have access to settings)

### Logs

None

**Comment by iBlazeX** [comment]
same here... did you find any solution?

**Comment by noobythedumbass** [comment]
@iBlazeX Yup, what I did was open command prompt by accessing task manager and running "cmd." Once command prompt was open, I typed in "sfc /scannow."

**Comment by Veneering7489** [comment]
> [@iBlazeX](https://github.com/iBlazeX) Yup, what I did was open command prompt by accessing task manager and running "cmd." Once command prompt was open, I typed in "sfc /scannow."

This command didn't work for me, what I did is type "DISM.exe /Online /Cleanup-image /Restorehealth", it shows "the requested operation failed. a system reboot is required to roll back change mode", and I ctrl+alt+delete reboot, then it shows updating, after login, it's back to normal.
https://support.microsoft.com/en-us/topic/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system-files-79aa86cb-ca52-166a-92a3-966e85d4094e

I'm scared by these system ricing tools now.

**Comment by Nimai-HK** [comment]
> > I'm scared by these system ricing tools now.
> 
> Me too.. I stopped using Rectify 11 after that. Now I customize everything one by one by myself with different tools. And my system is much stable now.

It's just that rectify 11 was designed for windows 11 (23h2 and below).. Microsoft brought a lot of changes in 24H2

The next version for rectify 11 (v4) which should support the updates is in beta rn
No ETA's as of now
could extend beyond late 2025.

**Comment by Nimai-HK** [comment]
> ### Description
> When I uninstalled Rectify11 using the installer and restarted my PC, the screen went black. I can still open apps using keyboard shortcuts, but running explorer.exe doesn't work. Can anyone help me?
> 
> ### Steps To Reproduce
> 1. Install Rectify11 on your PC.
> 2. Use the installer to uninstall Rectify11.
> 3. Restart your PC after the uninstallation.
> 4. Notice that the screen is black after restarting.
> 5. Try opening apps using keyboard shortcuts (observe that this sometimes works).
> 6. Attempt to run explorer.exe (observe that it doesn't work).
> 
> ### Rectify11 Version
> Version: 3.2
> 
> ### Windows Version
> Windows 11 24H2 (I don't know the build version since I don't have access to settings)
> 
> ### Logs
> None



Once you've reached this state,
I'd say enter the recovery environment and run system restore (in case you created any restore points before this failure)

This will roll back the OS to it's previous state (when rectify was working)

after rolling back, uninstall rectify 11 and run these commands in your cmd(admin) (one by one)

```
sfc /scannow
DISM /Online /Cleanup-Image /RestoreHealth
```

after it completes, restart your pc (❗Do not update and restart)

and then update windows and your drivers.


Rectify 11 v3.xis not built for Windows 11 24H2

**Comment by Nimai-HK** [comment]
If none of this works, try entering recovery terminal and offloading/copying your data to a USB stick using commands

and then reinstall windows 11 using another USB
On the USB you should download the ISO for your version from microsoft website (❗verify installation site)

Remember to run the correct index number in the same terminal command for your windows version
otherwise it could fail activation since your microsoft account might be linked to keys of another windows 11 version (As in Home, Pro, LTSc and so on)
